### DISCUSSION TITLE
QueryException: Detected an illegal character in input string

### DISCUSSION THREAD

**Discussion by lacerdanycolas** [laravel/framework#48429] [Q&A]
When making a call to the API I'm getting this error, and I don't understand what could be wrong, any ideas?

**Error when calling /listAllQuestions**
![image](https://github.com/laravel/framework/assets/18706924/9a67b778-3433-4c8a-8626-4214ed503730)

**api.php** 
![image](https://github.com/laravel/framework/assets/18706924/90c21444-d4b3-41e7-ae87-fae16942c7b7)

**QuestionsController.php**
![image](https://github.com/laravel/framework/assets/18706924/c90dcd7d-b501-44c5-a00a-6abd8606554d)

**Questions.php (model)**
![image](https://github.com/laravel/framework/assets/18706924/41719ed8-6a5c-4bca-b799-151c81c59025)

**Comment by Angel5a** [comment]
Looks like you are using non ASCII symbol in `created_at` or in `desc`.
